mne/utils.py

```diff
@@ -2144,6 +2144,19 @@ def _check_subject(class_subject, input_subject, raise_error=True):
         return None
 
 
+def _check_preload(inst, msg):
+    """Ensure data are preloaded."""
+    from .epochs import BaseEpochs
+
+    name = 'raw'
+    if isinstance(inst, BaseEpochs):
+        name = 'epochs'
```

Comment:
Bit late, but ...

`name = "epochs" if isinstance(inst, BaseEpochs) else "raw"` is more explicit IMO cause it makes it clear "name" is set there and exactly there